src/routes/backingImage/index.js

```
@@ -117,8 +117,11 @@ class BackingImage extends React.Component {
     } = this.props.backingImage
     const { backingImageUploadPercent, backingImageUploadStarted } = this.props.app
 
-    const defaultReplicaCount = settingData.find(s => s.id === 'default-replica-count')
-    const defaultNumberOfReplicas = defaultReplicaCount ? parseInt(defaultReplicaCount.value, 10) : 3
+    const settingsMap = settingData.reduce((acc, setting = {}) => ({ ...acc, [setting.id]: setting.value }), {})
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)